src/java.desktop/macosx/native/libawt_lwawt/awt/a11y/CommonComponentAccessibility.m

```diff
@@ -600,11 +600,9 @@ - (void)getActionsWithEnv:(JNIEnv *)env
     jobject axAction = (*env)->CallStaticObjectMethod(env, sjc_CAccessibility, jm_getAccessibleAction, fAccessible, fComponent);
     CHECK_EXCEPTION();
     if (axAction != NULL) {
-        jclass jc_AccessibleAction = NULL;
-        GET_CLASS(jc_AccessibleAction, "javax/accessibility/AccessibleAction");
-        jmethodID jm_getAccessibleActionCount = NULL;
-        GET_METHOD(jm_getAccessibleActionCount, jc_AccessibleAction, "getAccessibleActionCount", "()I");
-        jint count = (*env)->CallIntMethod(env, axAction, jm_getAccessibleActionCount);
+        DECLARE_STATIC_METHOD(jm_getAccessibleActionCount, sjc_CAccessibility, "getAccessibleActionCount", "(Ljavax/accessibility/AccessibleAction;Ljava/awt/Component;)I");
+        jint count = (*env)->CallStaticIntMethod(env, sjc_CAccessibility, jm_getAccessibleActionCount, axAction, fComponent);
+        CHECK_EXCEPTION();
```

Comment:
Do we need to call "(*env)->DeleteLocalRef(env, axAction);" in case of error in CHECK_EXCEPTION?

Reply:
I think we still reach the DeleteLocalRef that's further down at line 624